src/plugins/data/public/antlr/opensearch_ppl/code_completion.ts

```diff
@@ -35,16 +37,48 @@ export const getSuggestions = async ({
     const finalSuggestions: QuerySuggestion[] = [];
 
     if (suggestions.suggestColumns) {
-      finalSuggestions.push(...fetchFieldSuggestions(indexPattern));
+      finalSuggestions.push(...fetchFieldSuggestions(indexPattern, (f: any) => `${f} `));
     }
 
+    if (suggestions.suggestAggregateFunctions) {
+      finalSuggestions.push(
+        ...PPL_AGGREGATE_FUNTIONS.map((af) => ({
+          text: `${af}()`,
+          type: monaco.languages.CompletionItemKind.Function,
+          insertText: af + '(${1:expr}) ',
```

Comment:
not familiar with the syntax, does `${1:expr}` mean the first placeholder is `expr` when user chooses this option?

Reply:
Not even monaco has the syntax so I didn't know what it was myself, eventually coincidentally found it here: https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax.
This essentially allows the user to jump (with tab) to the placeholder, and it selects the placeholder so that the user can replace it with something else. Then, the user can jump to the end of the text.